src/policy/client_policy.rs

```diff
@@ -92,6 +99,7 @@ impl Default for ClientPolicy {
             use_services_alternate: false,
             thread_pool_size: 128,
             cluster_name: None,
+            buffer_reclaim_threshold: 8192,
```

Comment:
IMO this value is too small and will lead to memory thrashing for many users. In the Go client, the default is 64KiB. For a 16 node cluster with maximum number of default connections (256), it amounts to `16 x 256 x 64Kib = 256 MiB`, which is not an unreasonable amount. Keep in mind that Aerospike is a giant hash table in the end, and people end up putting a lot of data inside every record due to lack of normalization.

The most memory efficient way of keeping buffers without potential thrashing would be a multi-sized pool with idle timeouts, but that would be a bit more complicated and also could lead to potential lock contentions.

If you are concerned with the 64Kib size, 32KiB would also work.